### DISCUSSION TITLE
bypy seems completely broken already

### DISCUSSION THREAD

**Discussion by vientito** [houtianze/bypy#663] [General]
ForkPoolWorker-8: <E> [15:55:58] Full URL: https://qd.baidupcs.com/rest/2.0/pcs/file?method=rapidupload&path=%2Fapps%2Fbypy%2Fmedia%2F%E3%80%90%E5%A8%9B%E6%A8%82%E7%84%A6%E9%BB%9E%E3%80%91Mo%E7%88%B8%E7%BD%95%E6%9C%89%E5%8F%97%E8%A8%AA%EF%BC%8C%E4%BB%A5%E3%80%8C%E5%97%B01-12-05.m4a&content-length=2489546&content-md5=0937c1d93l85975a56a88929ee1815f9&slice-md5=1858ef40be5acedfb4d4bc597731a749&content-crc32=0x986fe4a0&ondup=overwrite&access_token=121.49c347eb230ef9deb3273fd9b60c4567.Y_R3EcHxZAGlIqONQeVA2hKd2u-1vPYC9dvxtOS.mY6fQA
ForkPoolWorker-8: <E> [15:55:58] HTTP Response Status Code: 400
ForkPoolWorker-8: <E> [15:55:58] Error JSON returned:{'error_code': 31023, 'error_msg': 'param error', 'request_id': 7877763479938541695}
Error code: 31023
Error Description: param error
ForkPoolWorker-8: <E> [15:55:58] Website returned: {"error_code":31023,"error_msg":"param error","request_id":7877763479938541695}
ForkPoolWorker-8: <E> [15:55:58] Waiting 10 seconds before retrying

This has been going on for over 2 weeks now. Nothing gets uploaded. Always returns "param error with http 400"

**Comment by houtianze** [comment]
Thanks for the feedback, this is similar to #664 . It seems that Baidu changed (or dropped) the `rapideupload` api parameter and it always results in this error. I've made some changes and the latest version `1.8.4` should have resolved this.
